provided-template.html

```diff
@@ -0,0 +1,453 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+ <head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>Palo Alto United School District Newsletter</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://opensource.keycdn.com/fontawesome/4.7.0/font-awesome.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Fira+Sans" rel="stylesheet">
+</head>
+  <body style="margin: 0; padding: 0;background-color:#EEEEEE;font-family: 'Fira Sans', sans-serif;">
+    <!-- Styles used through the page-->
+    <style>
```

Comment:
Style tags should always provide a 'type' attribute. e.g. `<style type="text/css">`